apiserver/pkg/versioneddataset/versioned_dataset.go

```diff
@@ -125,11 +125,13 @@ func VersionFiles(ctx context.Context, c dynamic.Interface, input *generated.Ver
 	})
 
 	result := make([]generated.PageNode, 0)
+	var lastModifiedTime time.Time
```

Comment:
Thank you for your contribution. There is still some small problem here, the variable `lastModifiedTime` is defined outside the for loop, so its address is also unchanged. It can be changed as follows:

```go
for _, obj := range objectInfoList {
  lastModifiedTime := obj.LastModified
}
```

Reply:
okk @0xff-dev thank you for your insights i will look into it